go/cmd/vtctldclient/command/vreplication/movetables/create.go

```diff
@@ -40,6 +40,8 @@ var (
 		NoRoutingRules      bool
 		AtomicCopy          bool
 		WorkflowOptions     vtctldatapb.WorkflowOptions
+		// This maps to a WorkflowOptions.StripShardedAutoIncrement ENUM value.
+		StripShardedAutoIncrementStr string
```

Comment:
Along with the proto name change, we should change this name. It is confusing to call it `Strip` when it could be NONE/REMOVE/REPLACE.

Reply:
There seems to be some nuance here that I missed during review. So maybe you just need to explain what I'm missing :)
Quoting Rohit:
> Maybe consider naming the StripShardedAutoIncrement differently for the string and enum representations. Also adding a comment to note that the first one is from the CLI flags and second to set in the _vt.vreplication row.

Reply:
I tried to address this here: https://github.com/vitessio/vitess/pull/16860/commits/733198b0c7bbc7368a5ce007b28a87890fbb0f2e